.eslintignore

```diff
@@ -0,0 +1,5 @@
+viewer
+dist
+lib
+
+webpub-rnow/
```

Comment:
Are these included in the gitignore as well? If so I don't think you need them here just fyi. Not a big deal tho